docs/site/tutorials/todo-list/todo-list-tutorial-repository.md

```diff
@@ -85,6 +85,133 @@ export class TodoListRepository extends DefaultCrudRepository<
 }
 ```
 
+### Inclusion of Related Models
+
+To get the related `Todo` object for each `TodoList`, we have to override the
+`find` and `findById` functions.
+
+First add the following imports:
+
+```ts
+import {Filter, Options} from '@loopback/repository';
+import {TodoListWithRelations} from '../models';
+```
+
+Add the following two functions after the constructor:
+
+{% include code-caption.html content="src/repositories/todo-list.repository.ts" %}
+
+```ts
+async find(
+  filter?: Filter<TodoList>,
+  options?: Options,
+): Promise<TodoListWithRelations[]> {
+  // Prevent juggler for applying "include" filter
+  // Juggler is not aware of LB4 relations
+  const include = filter && filter.include;
+  filter = filter && Object.assign(filter, {include: undefined});
```

Comment:
Please don't modify the original filter object, it can lead to surprising behaviors e.g. when a test calls `find` two times with the same `filter` object.

```suggestion
  filter = Object.assign({}, filter, {include: undefined});
```

Or even a more modern flavor:

```suggestion
  filter = {...filter, include: undefined};
```